packages/brotli/brotli.1.2.0/opam

```diff
@@ -0,0 +1,58 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+homepage: "http://hyegar.com"
+bug-reports: "https://github.com/fxfactorial/ocaml-brotli/issues"
+license: "BSD-3-clause"
+tags: ["clib:stdc" "clib:brotli"]
+dev-repo: "https://github.com/fxfactorial/ocaml-brotli.git"
+build: [
+  ["oasis" "setup" "-setup-update" "dynamic"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+build-doc: ["ocaml" "setup.ml" "-doc"]
+remove: ["ocamlfind" "remove" "brotli"]
+depends: [
+  "oasis" {build}
+  "conf-autoconf" {build}
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+]
+depexts: [
+  [["debian"] ["clang" "libc++-dev"]]
+  [
+    ["linux" "osx" "source"]
+    [
+      "https://raw.githubusercontent.com/fxfactorial/ocaml-brotli/master/prepare.sh"
```

Comment:
That's sad. I really encourage you to take a look at google/brotli#326 and related PRs/features and perhaps submit a PR to homebrew to update the `brotli` package to also install headers and shared libraries. Another option is to bundle the brotli source code with your package and build and link with it which would work on OS X or Linux. Either of these options is much more preferable to having `source` depexts which are generally regarded as dangerous and unfriendly to users. Finally, simply ignoring the depexts system and including a build failure message telling users they need to install brotli libraries (and where to get them) would also be fine.